test/langtools/tools/javac/tree/NestedTypeToString.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2022, Google LLC. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
```

Comment:
The fix for JDK-8281238 required several existing tests to be updated, so it did not have a dedicated test written for it. For this follow-on fix, if the test is going to use an annotation processor, which is quite reasonable, I suggest:
* placing the test somewhere under test/langtools/tools/javac/processing/model
*  subclassing JavacTestingAbstractProcessor
*  using string-valued annotations to hold the expected result and checking them in the process method

There are other tests using processor following this model under test/langtools/tools/javac/processing/model.

Reply:
Done, thanks for taking a look!

Reply:
@jddarcy is there any other feedback I can address?